js/ui/control/geolocate.js

```diff
@@ -94,6 +94,6 @@ Geolocate.prototype = util.inherit(Control, {
  * @event error
  * @memberof Geolocate
  * @instance
- * @property {EventData} data The returned PositionError object from the callback in [Geolocation.getCurrentPosition()](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/getCurrentPosition)
+ * @property {PositionError} data The returned [PositionError](https://developer.mozilla.org/en-US/docs/Web/API/PositionError) object from the callback in [Geolocation.getCurrentPosition()](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/getCurrentPosition).
```

Comment:
@lucaswoj For context: I added these links because the documentation generator does not understand those particular data types.